### PR TITLE
[docs][ybm] Edits to YBM PSE guide

### PR DESCRIPTION
**Pull request by ddhodge**
Edits to YBM PSE guide

@netlify /preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md [hunk 1]
```diff
@@ -12,15 +12,29 @@ menu:
...
+A private service endpoint (PSE) is used to connect a YugabyteDB Managed cluster that is deployed in a VPC with other services on the same cloud provider - typically a VPC hosting the application that you want to access your cluster. The PSE on your cluster connects to an endpoint on the VPC hosting your application over a privately linked service. 
 
 Setting up a private link to connect your cluster to your application VPC involves the following tasks:
 
-- Deploy your cluster in a VPC. You must create a VPC and deploy your cluster before you can configure a PSE.
-- Create a PSE in each region of your cluster. In the case of AWS, you grant access to one or more security principals in the form of Amazon resource names (ARNs); for Azure, you provide the subscription IDs of the services you want to have access.
-- Create a private endpoint on the cloud provider VPC (VNet).
+1. Deploy your cluster in a VPC. You must create a VPC and deploy your cluster before you can configure the PSE.
+1. Create a PSE in each region of your cluster. The PSE is an endpoint service, and you activate it by granting access to a service principal on your application VPC.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md:20]: what is a service principal?

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md [hunk 2]
```diff
@@ -12,15 +12,29 @@ menu:
...
 
-For more information on how to connect your cluster to an application over a private link using endpoints, refer to [Set up private link](../managed-endpoint-aws/).
+    In the case of AWS, a security principal is an AWS principal, in the form of Amazon resource names (ARNs).
+
+    For Azure, a security principal is a subscription ID of the service you want to have access.
+
+1. On the cloud provider, create an interface VPC endpoint (AWS) or private endpoint (Azure) on the VPC (VNet) hosting your application. You create an endpoint for each region in your cluster.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md:26]:
```suggestion
1. On the cloud provider, create an interface VPC endpoint (AWS) or a private endpoint (Azure) on the VPC (VNet) hosting your application. You create an endpoint for each region in your cluster.
```

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md [hunk 3]
```diff
@@ -39,3 +53,23 @@ To use ybm CLI, you need to do the following:
 
 - Create an API key. Refer to [API keys](../../../managed-automation/managed-apikeys/).
 - Install and configure ybm CLI. Refer to [Install and configure](../../../managed-automation/managed-cli/managed-cli-overview/).
+
+Note that, unlike VPC peering, when connected to an application VPC using a private link, you do not need to add an IP allow list to your cluster.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md:57]: link to IP allow list.

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md [hunk 4]
```diff
@@ -39,3 +53,23 @@ To use ybm CLI, you need to do the following:
...
+
+## Get started
+
+{{< sections/2-boxes >}}
+  {{< sections/bottom-image-box
+    title="Set up PrivateLink on AWS"
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md:63]: Either change to "Set up AWS PrivateLink" or "Set up private link on AWS". I get it that AWS and Azure conventions are different , but in our docs , the left nav heading is different and then the tile heading is different and it sorts of seems confusing.

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ Make sure that default security group in your application VPC allows internal co
 
 Make sure that the **Enable DNS resolution** and **Enable DNS hostnames** DNS settings in your application VPC are enabled. To access these settings, in the AWS [VPC console](https://console.aws.amazon.com/vpc/), select the VPC, click **Actions**, and choose **Edit VPC Settings**.
 
-To use AWS PrivateLink to connect your cluster to a VPC in AWS that hosts your application, first create a private service endpoint (PSE) on your cluster, then create an endpoint in AWS.
+To use AWS PrivateLink to connect your cluster to a VPC in AWS that hosts your application, first create a private service endpoint (PSE) for each region on your cluster, then create corresponding interface VPC endpoints in AWS.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md:46]:
```suggestion
To use AWS PrivateLink to connect your cluster to a VPC in AWS that hosts your application, first create a private service endpoint (PSE) for each region of your cluster, then create corresponding interface VPC endpoints in AWS.
```

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md [hunk 6]
```diff
@@ -211,9 +213,9 @@ aws ec2 describe-subnets \
     --query "Subnets[*].SubnetId" 
 ```
 
-This command returns the subnet IDs of the of the application VPC (`subnet_ids` in the following command). Note that the endpoint service can connect only from a subnet in the same availability zone as the PSE. If there isn't a subnet in the same zone, create one.
+This command returns the subnet IDs of the of the application VPC (`subnet_ids` in the following command). Note that the interface endpoint can connect only from a subnet in the same availability zone as the PSE. If there isn't a subnet in the same zone, create one.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/managed-endpoint-aws.md:216]:
```suggestion
This command returns the subnet IDs of the application VPC (`subnet_ids` in the following command). Note that the interface endpoint can connect only from a subnet in the same availability zone as the PSE. If there isn't a subnet in the same zone, create one.
```

**Review by aishwarya24 — COMMENTED** [review]
LGTM with some comments.

File: docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md [hunk 7]
```diff
@@ -12,16 +12,16 @@ menu:
 type: docs
 ---
 
-A private service endpoint (PSE) is used to connect a YugabyteDB Managed cluster that is deployed in a VPC with other services on the same cloud provider - typically a VPC that hosts an application that you want to have access to your cluster. The PSE connects to an endpoint attached to a VPC hosting your application over a privately linked service. Unlike VPC peering, when connected to a VPC using a private link, you do not need to add an IP allow list to your cluster.
+A private service endpoint (PSE) is used to connect a YugabyteDB Managed cluster that is deployed in a VPC with other services on the same cloud provider - typically a VPC that hosts an application that you want to have access to your cluster. The PSE on your cluster connects to an endpoint on a VPC hosting your application over a privately linked service. Unlike VPC peering, when connected to a VPC using a private link, you do not need to add an IP allow list to your cluster.
```
Comment (rangasayeelc-yb) [inline docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md]: modify "The PSE on your cluster connects to an endpoint on a VPC hosting your application over a privately linked service" 

to "The PSE on your cluster connects to an endpoint on a VPC hosting your application over a private connection"

**Pull request merged**